README.md

```diff
@@ -1 +1,19 @@
-# eq-schema-validator
\ No newline at end of file
+# eq-schema-validator
+An API for validating survey schemas.
+
+## Setup
+
+```
+brew install pyenv
+pip install --upgrade pip setuptools pipenv
+pipenv install
+pipenv install --dev
```

Comment:
do we need both `pipenv install --dev` and `pipenv install` i think `pipenv install --dev` includes both doesn't it?

Reply:
Ah I ran them the wrong way round and thought they were independently necessary.

Thanks, will update!